source/code-snippets/usage-examples/findOne.js

```diff
@@ -0,0 +1,22 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+      const query = { title: 'The Room' };
```

Comment:
I think a comment that gives additional context to this would be helpful. What is this query document?

e.g. does it find an exact match (Literally find this document `{ title: 'The Room' }`) or pattern match (it actually does this)